eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/IncrementalStake.hs

```diff
@@ -95,12 +95,25 @@ updateStakeDistribution ::
   IncrementalStake (EraCrypto era)
 updateStakeDistribution pp incStake0 utxoDel utxoAdd = incStake2
   where
-    incStake1 = incrementalAggregateUtxoCoinByCredential pp id utxoAdd incStake0
-    incStake2 = incrementalAggregateUtxoCoinByCredential pp invert utxoDel incStake1
+    incStake1 = incrementalAggregateUtxoCoinByCredential pp Id utxoAdd incStake0
+    incStake2 = incrementalAggregateUtxoCoinByCredential pp Invert utxoDel incStake1
+
+-- | Should a value be added (Id) or subtracted (Invert)
+data Mode = Id | Invert
```

Comment:
I love it that we no longer have to rely on negative coins :slightly_smiling_face: